### DISCUSSION TITLE
POIs

### DISCUSSION THREAD

**Discussion by TimWintle-Access** [albar965/littlenavmap#1080] [Q&A]
Hi Alex,

I was following a thread in which you advised that you were planning to add POI's to the littlenavmap DB in version 2.8.  Did there end up being a blocker to you being able to do this?  I can't see that it has ever been included in any releases and we're now up to version 3.x

Thanks
Tim

**Comment by albar965** [comment]
Not sure what you mean by "add POI's to the littlenavmap DB".

LNM has [Userpoints](https://www.littlenavmap.org/manuals/littlenavmap/release/latest/en/USERPOINT.html) for general use.
Regarding MSFS: There are already others providing this: [LittleNavMap 'Userpoint Icons'](https://flightsim.to/file/23181/littlenavmap-userpoint-icons).

And I did not plan to include POI databases in the LNM installation as a default.

**Comment by TimWintle-Access** [comment]
Thanks @albar965 

Apologies, I wasn't very clear.

Here's the original post I was referencing [https://www.avsim.com/forums/topic/597026-import-msfs-poi-to-little-navmap/?tab=comments#comment-4880118 ](https://www.avsim.com/forums/topic/597026-import-msfs-poi-to-little-navmap/?tab=comments#comment-4880118 )

I was wondering if you are able to extract the POI information directly from the database (when your tool breaks down the BGL files) so that rather than import the POI's and potentially make mistakes or miss some, they're all there in glorious technicolour with co-ordinates taken directly from the simulator its-self?  Asobo/ORBX have created many points of interest and I like to incorporate seeing them on my route.  At the moment, I do just import the userpoints as per [https://flightsim.to/file/17193/little-navmap-msfs-poi-database](https://flightsim.to/file/17193/little-navmap-msfs-poi-database)

Kind Regards
Tim

**Comment by albar965** [comment]
I see. Sorry but since is already provided by others I put this to low priority. I understand this does not help with Orbx POIs.

Maybe I can add this to 3.2 or one of the 3.0.X updates. No promises, though.

Alex
